project/S3Dependencies.scala

```diff
@@ -0,0 +1,17 @@
+import sbt._
+
+object S3Dependencies {
+
+  lazy val awsVersion = "2.17.287"
```

Comment:
This is in conflict with the `zioAwsVersion` below - zio-aws `5.17.280.1` depends on AWS Java SDK `2.17.280` while so it's going to be updated by this higher version here. I would only use the zio-aws version here and let it transitively bring in the corresponding AWS Java version.

Reply:
Updated.

Reply:
Made a ticket for the multiple regions suggestion 👍  - https://github.com/zio/zio-connect/issues/88